app/src/main/java/com/nextcloud/talk/users/UserManager.kt

```diff
@@ -43,6 +43,7 @@ class UserManager internal constructor(private val userRepository: UsersReposito
     val currentUser: Maybe<User>
         get() {
             return userRepository.getActiveUser()
+                .switchIfEmpty(getAnyUserAndSetAsActive())
```

Comment:
this should fix the app on phones that have an invalid state because of no users marked as "current".
:+1: or :-1: @AlvaroBrey ?

Reply:
Yeah, I don't see any problem with this.

However I will say that most classes should aim to use `CurrentUserProvider` instead,  which always holds a reference to the updated current user, without having to call the DB every time.

Reply:
:+1: but for now PR approved?

Reply:
I don't have the bandwidth to review the full PR right now, but as I said, no problem with this.